src/main/kotlin/io/github/anki/anki/controller/model/CardDto.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.anki.anki.controller.model
+import io.github.anki.anki.repository.mongodb.model.MongoCard
+import io.github.anki.anki.service.model.Card
+import org.bson.types.ObjectId
+
+
+class CardDto (
```

Comment:
`DtoCard`? like `MongoCard`
or change mongo card instead to `CardMongo`

Reply:
+ CardDto and MongoCard